tests/network/sriov.go

```diff
@@ -46,8 +47,6 @@ import (
 	"kubevirt.io/kubevirt/pkg/network/vmispec"
 	"kubevirt.io/kubevirt/pkg/util/hardware"
 	"kubevirt.io/kubevirt/pkg/util/net/dns"
-	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
-
```

Comment:
Separating between the tests dependencies and the project codebase is nicer IMO.